crawl-ref/source/mutation-data.h

```diff
@@ -1848,12 +1848,18 @@ static const mutation_def mut_data[] =
   {"You can once more drink potions while threatened.", "", ""},
 },
 
-{ MUT_READ_SAFETY, 10, 1, mutflag::bad | mutflag::xom, false,
+{ MUT_READ_SAFETY, 7, 3, mutflag::bad | mutflag::xom, false,
   "inability to read while threatened",
 
-  {"You sometimes lose the ability to read scrolls when taking damage.", "", ""},
-  {"You lose the ability to read scrolls when threatened.", "", ""},
-  {"You can once more read scrolls while threatened.", "", ""},
+  {"You rarely lose the ability to read scrolls when taking damage.",
+   "You ocassionally lose the ability to read scrolls when taking damage.",
+   "You sometimes lose the ability to read scrolls when taking damage."},
```

Comment:
Gammafunk and I suggest occasionally / sometimes / often as the adverbs here to match teleportitis adverbs.

Reply:
Good suggestion, my main thought was that increasing the frequency for the third tier might confuse players that the mutation effect is now more common. But since you suggest it, it works for me.